tutorials/plot_events_and_annotations.py

```diff
@@ -0,0 +1,120 @@
+"""
+The **events** and :class:`Annotations <mne.Annotations>` data structures
+=========================================================================
+
+Events and annotations are quite similar. This tutorial highlights their
+differences and similitudes and tries to shade some light to which one is
+preferred to use in different situations when using MNE.
+Here follows both terms definition from the glossary.
+
+    events
+        Events correspond to specific time points in raw data; e.g., triggers,
+        experimental condition events, etc. MNE represents events with integers
+        that are stored in numpy arrays of shape (n_events, 3). Such arrays are
+        classically obtained from a trigger channel, also referred to as stim
+        channel.
+
+    annotations
```

Comment:
Link to annotations object?

Reply:
It is linked everywhere, but here 'cos this is a citation of the glossary:
![image](https://user-images.githubusercontent.com/7044835/48313356-75a20e80-e5bb-11e8-8570-e5a5ecff928a.png)